### ISSUE TITLE
Multiply distances by 10 before NCA loss?

### ISSUE THREAD

**Issue by reppertj** [littleredxh/HardNegative#1]
I really enjoyed the paper, especially the visualizations that accompanied your argument. You made a pretty persuasive case about the dynamics of backprop on hard negative triplets.

I'm curious why, looking at the SCTLoss implementation, you multiply the distances by 10 here before the NCA loss; what was behind validating this scalar? I imagine it has something to do with putting the two loss functions on an even footing, but I can't quite see it.
https://github.com/littleredxh/HardNegative/blob/f5f392377fd0a83dd7e9ae590042960d40f6067c/_code/Loss.py#L104

Also, how did you decide on the 0.8 threshold for cosine similarity in your "selection" function? What tends to happen if you leave this out?
https://github.com/littleredxh/HardNegative/blob/f5f392377fd0a83dd7e9ae590042960d40f6067c/_code/Loss.py#L87-L88

Thanks—this is a great paper.

**Comment by reppertj** [comment]
Ah, of course, that's the temperature parameter!
